php/api.php

```diff
@@ -5,12 +5,6 @@
 # @copyright	2015 by Tobias Reich
 ###
 
-@ini_set('max_execution_time', '200');
```

Comment:
I like to keep configuration options to the user, the are recommended and should not be enforced